src/Components/SavedSearchAlert/SavedSearchAlertModal.tsx

```diff
@@ -203,21 +214,7 @@ export const SavedSearchAlertModal: FC<SavedSearchAlertFormProps> = ({
           values.email || values.push || values.hearFromArtsyAdvisor
 
         return (
-          <ModalDialog
-            onClose={onClose}
-            title="Create Alert"
-            data-testid="CreateAlertModal"
-            footer={
-              <Button
-                disabled={!hasSelectedContactMethod || !isPhoneValid}
-                loading={isSubmitting}
-                onClick={() => handleSubmit()}
-                width="100%"
-              >
-                Save Alert
-              </Button>
-            }
-          >
```

Comment:
We had to remove the Submit button from the Modal since the Modal has to wrap both components in order to transition between screens.

Reply:
No need to remove it if you don't want to remove it

Reply:
I misunderstood: yeah you basically have to build a custom dialog to do this feature. This is not unheard of in Force (see onboarding, for instance). But yeah, avoid fighting with the existing `ModalDialog` and just step completely out of it.